examples/ethernet/ethernet.ino

```diff
@@ -0,0 +1,49 @@
+#include <simpleRPC.h>
+#include <Ethernet.h>
+
+byte mac[] = { 0xDE, 0xAD, 0xBE, 0xEF, 0xFE, 0xED };
+IPAddress ip(192, 168, 1, 50);
+EthernetServer server(10000);
+
+byte ping(byte data) {
+  return data;
+}
+
+void connectEthernet() {
+  if (Ethernet.hardwareStatus() == EthernetNoHardware) {
+    while (true) {
+      delay(1);  // do nothing, no point running without Ethernet hardware
+    }
+  }
+}
+
+void printEthernetStatus(Print& print) {
+  print.print("IP Address: ");
+  print.println(ip);
+  if (Ethernet.linkStatus() == LinkOFF) {
+    print.println("Ethernet cable is not connected.");
+  }
+}
+
+void setup(void) {
+  Serial.begin(9600);
+  Ethernet.begin(mac, ip);
+  connectEthernet();
+  printEthernetStatus(Serial);
+  server.begin();
+}
+
+void loop(void) {
+  EthernetClient client = server.available();
+  if (client) {
+    while (client.connected()) {
+      if (client.available()) {
```

Comment:
I wonder if this line is needed, in `interface.tcc` on line 113, we seem to do the same thing.